src/frontend/src/pages/GanttPage/GanttChart.tsx

```diff
@@ -4,29 +4,21 @@
  */
 
 import { Typography } from '@mui/material';
-import { calculateEndDate } from 'shared';
-import { Gantt } from './GanttPackage/components/gantt/gantt';
-import { Task, ViewMode } from './GanttPackage/types/public-types';
+import { Gantt } from './Gantt';
+import { EventChange, Task } from '../../utils/gantt.utils';
 
 interface GanttPageProps {
   ganttTasks: Task[];
+  start: Date;
+  end: Date;
   onExpanderClick: (ganttTasks: Task) => void;
+  isEditMode: boolean;
+  saveChanges: (eventChanges: EventChange[]) => void;
 }
 
-const GanttChart: React.FC<GanttPageProps> = ({ ganttTasks, onExpanderClick }) => {
+const GanttChart: React.FC<GanttPageProps> = ({ ganttTasks, start, end, onExpanderClick, isEditMode, saveChanges }) => {
```

Comment:
I think this file could be integrated into the GanttChartSection Component, see my other comment